### PR TITLE
[docs] fix markdown issue in fragments.md

### PR DESCRIPTION
**Pull request by lucasweng**
Currently, this doesn't render right:

Screen Shot 2021-03-14 at 11 09 51 AM

The change:

The generated Flow types include a type for the fragment reference, which is the type with the `$key` suffix: `<fragment_name>$key`, and a type for the shape of the data, which is the type with the `$data` suffix: `<fragment_name>$data`; these types are available to import from files that are generated with the following name: `<fragment_name>.graphql.js`

### CODE REVIEW

File: website/versioned_docs/version-v11.0.0/guided-tour/rendering/fragments.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Let's distill what's going on here:
 * A *fragment reference* is an object that Relay uses to *read* the data declared in the fragment definition; as you can see, the `UserComponent_user` fragment itself just declares fields on the `User` type, but we need to know *which* specific user to read those fields from; this is what the fragment reference corresponds to. In other words, a fragment reference ** is like *a pointer to a specific instance of a type* that we want to read data from.
```
Comment (poteto) [inline website/versioned_docs/version-v11.0.0/guided-tour/rendering/fragments.md:85]: Do you mind removing these stray `**`s too?

File: website/docs/guided-tour/rendering/fragments.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Let's distill what's going on here:
...
 * Relay ** will automatically generate Flow types for any declared fragments when the compiler is run, so you can use these types to declare the type for your Component's `props`.
-    * The generated Flow types include a type for the fragment reference, which is the type with the `$key` suffix: ``*<fragment_name>*``$key``, and a type for the shape of the data, which is the type with the `$data` suffix:  ``*<fragment_name>*``$data``; these types are available to import from files that are generated with the following name: `*<fragment_name>*.graphql.js`.
+    * The generated Flow types include a type for the fragment reference, which is the type with the `$key` suffix: `*<fragment_name>*$key`, and a type for the shape of the data, which is the type with the `$data` suffix:  `*<fragment_name>*$data`; these types are available to import from files that are generated with the following name: `*<fragment_name>*.graphql.js`.
```
[the remark this answers is not in the document]
Comment (lucasweng) [reply website/docs/guided-tour/rendering/fragments.md:88]: Got it. I'll remove those `*`s and `**`s. Thought the `*<>*` was some sort of syntax highlight. 😅

File: website/versioned_docs/version-v11.0.0/guided-tour/rendering/fragments.md (hunk 1 above)
Comment (lucasweng) [reply website/versioned_docs/version-v11.0.0/guided-tour/rendering/fragments.md]: Thanks for pointing that out. All `*`s and `**`s are dropped! https://github.com/facebook/relay/pull/3408/commits/8b21be8616af5ebccaddcc50195bcf88a144ff0c

**Review by poteto — APPROVED** [review]
Thanks @lucasweng, LGTM!

**Pull request closed without merging**